packages/models/src/lib/implementation/utils.ts

```diff
@@ -6,6 +6,16 @@
  */
 export const slugRegex = /^[a-z0-9]+(?:-[a-z0-9]+)*$/;
 
+/**
+ * Regular expression for the default report filename.
+ */
+export const REPORT_NAME_PATTERN =
+  /^(report\.)(\d{4}-\d{2}-\d{2}T\d{2}-\d{2}-\d{2}\.\d{3}Z)/;
+export function reportNameFromReport<T extends { date: string }>(
+  report: T,
+): string {
+  return `report.${report.date.replace(/:/g, '-')}`;
+}
```

Comment:
The function name is pretty awkward, don't have to mention "report" twice. How about `reportFileName`?

Reply:
done